ciao-cli/node.go

```diff
@@ -123,6 +143,79 @@ func listComputeNodes(t *template.Template) error {
 	}
 	return nil
 }
+func listNetworkNodes(t *template.Template) error {
+	var nodes types.CiaoNodes
+
+	url := buildComputeURL("nodes/network")
+
+	resp, err := sendHTTPRequest("GET", url, nil, nil)
```

Comment:
The code from here to the end of the function is almost identical to the code in ListComputeNode and listNodes.  Maybe we could refactor.